PlaceRouteHierFlow/placer/ILP_solver.cpp

```diff
@@ -358,7 +358,11 @@ double ILP_solver::GenerateValidSolution(design& mydesign, SeqPair& curr_sp, PnR
 
   // set_add_rowmode(lp, FALSE);
   {
-    double row[N_var + 1] = {0};
+    double row[N_var + 1];
+    for (unsigned int i = 0; i<N_var+1; ++i) {
```

Comment:
{0} initializes the entire array to zeroes . its the universal zero-initializer.

Reply:
i changed this because it didn't compile using clang++-10 
These arrays on the stack I think are the culprit for the Release buildtype issues.